src/components/Collective/CollectiveActions.vue

```diff
@@ -43,7 +43,7 @@
 		<NcActionSeparator v-if="collectiveCanShare(collective) && !isPublic" />
 		<NcActionLink :close-after-click="true"
 			:href="printLink"
-			target="_blank">
+			target="ncCollectivesPrint">
```

Comment:
Not sure what's the effect here? :thinking: Also, for me (with Firefox) this results in an error: `[Vue warn]: Invalid prop: custom validator check failed for prop "target".`

Reply:
The effect should be that the new tab / window is reused, `_blank` always opens a new window, but if you give a unique string, then that tab is used for all `target="myName"` links.
See MDN on this: [developer.mozilla.org#reuse_existing_windows_and_avoid_target_blank](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#reuse_existing_windows_and_avoid_target_blank)

But it looks like the `NcActionLink` has a custom validator only allowing `_...`, so this needs to be fixed in the vue components library. (That error is raised for development build).

Reply:
Ah, that makes a lot of sense. Thanks for the explanation :heart: